int_experian/int_experian_sfra/cartridge/static/default/js/edqUtils.js

```diff
@@ -458,6 +463,17 @@ function edqSetGlobalIntuitiveConfiguration() {
 		},
 	];
 }
+function setCountryField() {
+	setTimeout(function() {
```

Comment:
Why is `setTimeout` necessary?

Reply:
Sometimes the function run really quick and do the check up before the "edq-metadata" attribute is set, even if the user has selected one option on the suggested address box.